File: Java/hw1.java
import java.io.BufferedReader;
import java.util.*;
import java.io.FileReader;
import java.io.PrintWriter;

public class hw1 {
	
	public static void hw1(String inFile,String outFile)throws Exception{
		FileReader file=new FileReader(inFile);//"C:/Users/adity/Desktop/test.txt");
		BufferedReader reader=new BufferedReader(file);
		String text="";
		String newline = System.getProperty("line.separator");
		String line=reader.readLine();
		PrintWriter writer = new PrintWriter(outFile, "UTF-8");			
		while(line!=null){
			char cur=97;
			boolean value=true;
			int count=0;//26 total
			text=line;
			for (int i = 0 ; i != text.length() ; i++) {
			    for(int j=0;j!=text.length();j++){
				char c = text.charAt(j);
				if(c==cur){
			    count++;cur+=1;	
			    break;
			    }			    			   
			    }
			}
			if(count==26){value=true;}
			else if (count!=26){value=false;}
			writer.println(value);
			line=reader.readLine();
			}
		writer.close();
	}
}

File: Python/hw1.py
def hw1(infile,outfile):
    f=open(infile,'r')
    f2=open(outfile,'w')
    for line in f:
        cur='a'
        value='true'
        count=0
        text=line
        b=list(text)
        for i in range(0,len(text)):
            for j in range(0,len(text)):
                if cur==b[j]:
                    count=count+1
                    cur=chr(ord(cur)+1)
                    break
        if(count==26):
            value='true'
        elif(count!=26):
            value='false'
        f2.write(value+'\n')
    f2.close()
    f.close()
